### PR TITLE
The Shovellening: Coroner and Miner Coats can now hold Shovels

### PR DESCRIPTION
**Pull request by DrAmazing343**
## About The Pull Request
The Shovel, Serrated Bone Shovel, and the Entrenching Tooll can now be worn on the Coroner/Miner Winter Coats, as well as upon the Coroner's labcoat. Normally, shovels worn on the belt slot are displayed on your sprite at your back, and allowing them to go on the suit slot initially mirrored them to the front on side-facing mobs. 

I've changed the mirrored sprites for side-facing in the belt_mirror.dmi in order to _not_ be mirrored, so they display properly on the back. Not a very elegant way to do it, but this hasn't caused issues with double-equipping shovels in testing.

shovel_example

## Why It's Good For The Game
Miner Winter Coats should be able to hold shovels as they're mining tools to begin with, and Coroners should be able to do the same in order to dig or dig up graves. The Serrated Shovel, as well, is the only Morbid Bone Saw equivalent in the game as far as I know, so this would allow Coroners to carry that Morbid equivalent without taking up bag space. This encourages Coroners to lean further into the flavor of their role as Morbid or Occultic Medbay personnel.

## Changelog

:cl: Jane
qol: Shovels, Serrated Bone Shovels, and Entrenching Tools can now be hung upon Miner and Coroner Winter Coats, as well as the Coroner's Labcoat!
image: Mirrored the mirrored sprites to display Shovels in the suit slot properly.
/:cl:

### CODE REVIEW

**Comment by EricZilla** [comment]
Let's goooooo

**Comment by Wallemations** [comment]
Make sure this includes the market gardener

**Comment by DrAmazing343** [comment]
> Make sure this includes the market gardener
[https://cdn.discordapp.com/attachments/704897260957335662/1237610616206921819/image.png?ex=663c4634&is=663af4b4&hm=4dc4e2dbc3318f96642bd9b6e0e106571fe9e1c7446a2d4ddb72ea9ad5dc0591&](url)
Done and done, just figuring out how to make it not force a visual we don't have. The Entrenching Tool doesn't have a belt sprite because it's not beltable, but the game wants to call it a belt and call it a day.

**Comment by DrAmazing343** [comment]
And as simple as a blank .dmi addition, it's added. Anything else?

**Comment by Dawnseer** [comment]
I'm just thinking, wouldn't it fit on the chaplain's clothes too? Let the preacher go out to dig graves. Then let the coroner go dig them up later

**Comment by DrAmazing343** [comment]
> I'm just thinking, wouldn't it fit on the chaplain's clothes too? Let the preacher go out to dig graves. Then let the coroner go dig them up later

As much as I'd like to put this into practice, the wide variety of Chaplain garments makes it a big undertaking unless I only put it on a few specific pieces of clothing. As it stands, Chaplains can wear shovels on their belts as anyone can, and unless someone wants to draw up a discrete list, I'll probably leave this PR as-is for now.

File: code/modules/clothing/suits/wintercoats.dm [hunk 1]
```diff
@@ -629,6 +632,9 @@
 		/obj/item/storage/bag/ore,
 		/obj/item/t_scanner/adv_mining_scanner,
 		/obj/item/tank/internals,
+		/obj/item/shovel,
+		/obj/item/shovel/serrated,
```
Comment (Ghommie) [inline code/modules/clothing/suits/wintercoats.dm:636]:
```suggestion
		/obj/item/shovel,
```
The compile-time list of allowed suit-slotted items is typecached on runtime iirc.

**Pull request merged**